### ISSUE TITLE
Releases on DockerHub

### ISSUE THREAD

**Issue by ricschuster** [NCC-CNC/wheretowork#228]
What do you think about starting to create releases on DockerHub for updates of Where To Work?
The 'projection on the fly' and the 'mapping' updates would make for reasonable releases I think.
Maybe 'projection on the fly' is too granular, but thinking about releases a bit might be useful.

**Comment by jeffreyhanson** [comment]
Yeah, using fixed releases would help provide a more stable user experience. E.g., if you're worried about updates/bugs causing downtime or bad experiences for users? The flipside is that users won't get to experience newer updates as quickly. It would be possible to set up a ShinyProxy so that beginner users can only see the "released" versions and advanced (or what ever user) can see the latest (aka "nightly") builds?

**Comment by ricschuster** [comment]
Thanks Jeff!
I was thinking about a form of version tracking to make sure we could provide a stable user experience. Maybe I am overthinking this and the current system is fine, but I thought I'd mention it.

**Comment by jeffreyhanson** [comment]
I think there's pro's and con's - I'm not sure what the best strategy would be. @DanWismer, do you have any thoughts on this?

**Comment by DanWismer** [comment]
Are we thinking of implementing what we did for the `impact (project evaluation tool)` app? Where the `dev` tag listens to the developer branch and the `latest` tag listens to the production branch? Or rather are releases a form of archiving?

**Comment by ricschuster** [comment]
Good question. I do like the `dev` and `latest` tag component. I was mostly thinking about archiving initially, but don't have a strong opinion about this.

**Comment by DanWismer** [comment]
Under the "Builds" tab on docker hub, I see that there is a collection of recent builds (some type of cache?). They are not named well, but could we re-name one of the previous builds we would consider as stable? That way, if I merge something that breaks production, we will have a safety net to pull a stable build back to the server.
